next.config.js

```diff
@@ -6,7 +6,8 @@ const bundleAnalyzer = require('@next/bundle-analyzer')({
 const nextTranslate = require('next-translate');
 
 const nextConfig = {
-  reactStrictMode: true,
+  // Temporary set to `false` to disable mismatch `id` error caused by Radix UI
+  reactStrictMode: false,
```

Comment:
If the error is only logged in the console, I don't think disabling this is necessary. We may miss other bugs.

![image](https://user-images.githubusercontent.com/52666982/145433582-37fa08bc-7107-4a05-ba47-0b3b422ff18d.png)

Reply:
Waking up today and realizing this can also be fix by only render `Menu` on client-side temporarily until this is fixed

Reply:
But why, it doesn't affect anything right? I have seen that error countless of times, the error was for something like aria-invalid="true" on server but "false" on client. Id doesn't matter, as long as on the client all the related components have the same it.

Reply:
The reason is on the server they run the generator for id, and on the client they run it again. Id is randomized everytime so they are different. I solved this for useUuid by only generating the ids on client, but in the end it doesn't matter.

Reply:
I understand why this happens. In fact React 18 will be released with a `useId()` hook to solve this issue but the Radix is just gonna release its own custom hook in adnavce to solve it.

As for why, it's simply anoying. Let's do nothing then (besides enable reactStrictMode)